frontend/components/Nav/ParentNav/ParentNav.jsx

```diff
@@ -1,14 +1,13 @@
 'use client';
-import { useContext } from 'react';
-import { TeamsContext } from '@app/teams/layout';
 import LogoutButton from '../LogoutButton';
 import NavLink from '../NavLink';
 import SubNav from '../SubNav/SubNav';
 import navStyle from './ParentNav.module.css';
+import { useUser } from '@user-context';
 
 const ParentNav = () => {
-  const { teamList } = useContext(TeamsContext);
-
+  const { user } = useUser();
```

Comment:
I thiiiiink (but not 100% sure) you can also destructure the array to clean up these 2 lines.

```
  const { user: [ teams] } = useUser();
```